tools/github-actions/audit/src/main.ts

```diff
@@ -5,54 +5,148 @@ import {GitHubAdvisoryId, NPMAuditReportV1, Severity} from 'audit-types';
 import Audit = NPMAuditReportV1.Audit;
 import Advisory = NPMAuditReportV1.Advisory;
 
+/**
+ * Severities supported by yarn npm audit from the lowest to the highest criticality
+ */
+const severities: Severity[] = ['info', 'low', 'moderate', 'high', 'critical'];
+const colors = ['', 'green', 'yellow', 'orange', 'red'];
+
+/**
+ * Interface to describe Yarn 4+ npm audit response.
+ * It is not yet covered by audit-types
+ */
+interface Yarn4AuditResponse {
```

Comment:
Do you know it this types can be exposed by [@yarnpkg/types](https://www.npmjs.com/package/@yarnpkg/types) ?

Reply:
Nope, it seems more linked to methods and types around workspace management etc. there is nothing around audit on this one. I'll try to check in yarnpkg scope if there is anything exposed

Reply:
Nope I didn't find anything relevant